elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -35,7 +40,12 @@ $(function() {
             </ul>
             <ul id="order-blocks-dropzone-{{uuid}}" name="{{answer_name}}" class="list-group pl-order-blocks-connected-sortable dropzone">
                 {{#submission_dict}}
-                    <li uuid="{{uuid}}" string="{{inner_html}}" style='margin-left: {{indent}}px;' class="list-group-item shadow">{{{inner_html}}}</li>
+                    {{#code}}
+                    <li class="shadow" uuid="{{uuid}}" style="list-style-type:none;" string="{{inner_html}}" style='margin-left: {{indent}}px;><pl-code language="{{code_language}}">{{{inner_html}}}</pl-code></li>
+                    {{/code}}
+                    {{^code}}
+                    <li class="list-group-item shadow" uuid="{{uuid}}" string="{{inner_html}}" style='margin-left: {{indent}}px;>{{{inner_html}}}</li>
+                    {{/code}}
```

Comment:
I'm not really a fan of all the extra complexity added to this file, but I'm not sure if there is a better way.

Reply:
I've fixed it by just adding the `pl-code` tags in the prepare function. This also fixed my other bug!